lib/liquid/combinators/general.ex

```diff
@@ -242,15 +239,36 @@ defmodule Liquid.Combinators.General do
   """
   def variable_name do
     parsec(:variable_definition)
-    |> tag(:variable_name)
+    |> unwrap_and_tag(:variable_name)
   end
 
-  def liquid_variable do
+  def quoted_variable_name do
+    parsec(:ignore_whitespaces)
+    |> ignore(utf8_char([@single_quote]))
```

Comment:
Could you avoid repetition of `utf8_char([@single_quote])`

Reply:
Yes, I could create a function which returns `utf8_char([@single_quote])` but redundancy is minimal (only appears four times) and here is explicit that I want use the utf8 of single_quote

Reply:
Understood.